### DISCUSSION TITLE
InfluxDB v1 and Grafana are now deprecated

### DISCUSSION THREAD

**Discussion by jertel** [jertel/vuegraf#114] [General]
With InfluxDB v1's last release arriving almost a year and a half ago, it's time to look beyond Influx v1. Vuegraf has long supported InfluxDB 2, but until today hasn't provided a compatible dashboard or related documentation. That's no longer the case, as this project now includes a fully operational dashboard utilizing the newer Flux queries. Further, this new dashboard was written for Influx's native UI, which means that Grafana is no longer required. Consequently, I've removed documentation and dashboard template files for Grafana, as they are based on the obsolete Influx v1 query language.

The project [readme](https://github.com/jertel/vuegraf#upgrading-from-influxdb-v1) now provides upgrade instructions for those with InfluxDB v1 containers that would like to move forward with InfluxDB 2. This is a relatively low effort process and handles large volumes of data in short time. Please refer to the updated documentation for more information.

Note that this announcement is primarily relating to the project documentation and dashboard template update. I will leave the Influx v1 code in place for a while longer to make it easier for users who work through the InfluxDB upgrade process. Again, the actual Vuegraf Python code has not been modified at this time.

**Comment by dalklein** [comment]
I use IOTstack with Influx 1.8 and Grafana for other solar & battery controls.   I'm thinking that I'd like to use vuegraf to influxDB v1 as that's what I have already going.   Is there much difference about how vuegraf sends data into influx 1 vs. influx 2?  Anything I need to be careful of?   Thansk!   (and thanks for vuegraf)

**Reply by dalklein** [reply]
in vuegraf.py, it looks like it writes to which ever version of influxDB is connected.  That seems great for my use, I just want to get the data into the db, and I'd then plot & view it with Grafana.

**Reply by jertel** [reply]
Yes, it should still work. I no longer test v1 so there's always a chance a future release could break something, but it certainly wouldn't be deliberate. So think you'll be fine if you try it.

**Reply by dalklein** [reply]
Great!
